app/views/entities/new.html copy.erb

```diff
@@ -0,0 +1,43 @@
+<main class="body-container">
+  <section class="container">
+    <%= form_for [current_user, @group, @entity] do |f| %>
+      p "@entity"
+      p @entity
+      <% if @entity.errors.any? %>
+        <div id="error_explanation">
+          <h2><%= pluralize(@entity.errors.count, "error") %> prohibited this transacction from being saved:</h2>
+          <ul>
+            <% @entity.errors.full_messages.each do |message| %>
+              <li class="alert-danger"><%= message %></li>
+            <% end %>
+          </ul>
```

Comment:
[OPTIONAL]As the previous code reviewer request please kindly implement this change I know it an optional change but it should not be overlooked because it will help make your code better.

- Kindly make add the "Back" button (<) on the Transactions page, so that When the user clicks on the "Back" button the user navigates to the home page.

![236657956-bcca9520-e88a-4759-ad5d-aad2370c3cae](https://user-images.githubusercontent.com/79506105/236659218-aae893ba-df1c-4cf6-96f7-73269c775e59.png)

Your design should look like this:

![Screenshot 2023-05-07 at 7 25 04 AM](https://user-images.githubusercontent.com/79506105/236659290-4b0b8f97-e89e-4cc6-9b8c-90eff79311c2.png)

it is important to pay attention to detail follow design requirements